tests/integration/test_rest_handler_handler.py

```diff
@@ -48,6 +50,84 @@ def _cookie_header():
     return f"splunkd_8000={session_key}"
 
 
+def test_basic_crud_operations():
```

Comment:
Can you please also outline in the description of the test which actual files in `splunktaucclib` are being tested?